dango/types/src/oracle/price_source.rs

```diff
@@ -19,4 +27,22 @@ impl PriceSource {
             },
         }
     }
+
+    /// Does a raw query to the oracle contract to get the price for this price source.
+    pub fn raw_query_price(
+        &self,
+        querier: &QuerierWrapper,
+        oracle: Addr,
+    ) -> anyhow::Result<PrecisionedPrice> {
+        match self {
+            Self::Pyth { id, precision } => {
+                let price = querier
+                    .query_wasm_raw(oracle, PRICES.path(*id))?
+                    .ok_or(anyhow::anyhow!("Price not found for denom: {}", id))?
```

Comment:
Import `anyhow::anyhow` and invoke it as `anyhow!` here instead of `anyhow::anyhow!`